tests/bearlib/aspects/InstanceTest.py

```diff
@@ -31,12 +31,13 @@ def test_tastes(
 
     def test__setattr__(self, SubAspect, SubAspect_tastes):
         aspect = SubAspect('py')
+        error_taste = "A 'taste' value for this aspectclass instance "\
+            'exists already.'
+        error_aspectless = "can't set attributes of aspectclass instances"
```

Comment:
unused.

Reply:
Actually, I squashed an additional commit by mistake which did the overwriting on this commit. I've rectified it.

Thanks for the quick review @jayvdb! :-)